pytest.ini

```diff
@@ -2,3 +2,9 @@
 testpaths = kobo kpi hub
 env =
     DJANGO_SETTINGS_MODULE=kobo.settings.testing
+addopts =
+    -m "not performance"
+markers =
+    [
+        performance: marks test for performance that may unexpectedly fail on slower platforms (not executed by default - select with '-m "performance"')
+    ]
```

Comment:
What about moving the entire contents of this file into `pyproject.toml` instead of adding these settings twice? [and then deleting `pytest.ini`]